types/Shipment/Options/PrintCustomCode.d.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ *  - FedEx
+ *    - (null) - If print_custom_1_code is not provided it defaults to Customer Reference
+ *    - PO - Purchase Order Number
+ *    - DP - Department Number
+ *    - RMA - Return Merchandise Authorization
+ *  - UPS
+ *    - AJ - Accounts Receivable Customer Account
+ *    - AT - Appropriation Number
+ *    - BM - Bill of Lading Number
+ *    - 9V - Collect on Delivery (COD) Number
+ *    - ON - Dealer Order Number
+ *    - DP - Department Number
+ *    - 3Q - Food and Drug Administration (FDA) Product Code
+ *    - IK - Invoice Number
+ *    - MK - Manifest Key Number
+ *    - MJ - Model Number
+ *    - PM - Part Number
+ *    - PC - Production Code
+ *    - PO - Purchase Order Number
+ *    - RQ - Purchase Request Number
+ *    - RZ - Return Authorization Number
+ *    - SA - Salesperson Number
+ *    - SE - Serial Number
+ *    - ST - Store Number
+ *    - TN - Transaction Reference Number
+ *    - EI - Employer's ID Number
+ *    - TJ - Federal Taxpayer ID No.
+ */
+export declare type TPrintCustomCode =
+  | 'PO'
+  | 'DP'
```

Comment:
**Question:**
Since the values are but obscure here, do you think it would be worth turning this into an enum?
```typescript
export enum EPrintCustomCode {
  PurchaseOrderNumber = "PO",
  DepartmentNumber = "DP",
  ...
}
```

Reply:
That's a great idea. I assume then you would just swap out `TPrintCustomCode` references for the new `EPrintCustomCode` meaning it would have to be one of the enums? We don't have any other enum use in the lib at present to base it off of.

Reply:
Yea, in general, you would just use the string union as it removes some overhead, but in cases like this where its not obvious what the strings represent, I like enums, because you can use it just like a string union type (yea just swap `TPrintCustomCode` with `EPrintCustomCode`), but then if you want to use this, you can just import it.
```typescript
import { EPrintCustomCode } from "@easypost/api";
// ...
someFunction(EPrintCustomCode.DepartmentNumber)
```

But it is effectively the same as the string union type, and this is just a suggestion so if you don't think its worth it, thats fine too.

Reply:
Right. After looking this over, we actually accept `null` on this field which sets it to something. In favor of breaking that behavior, I'm inclined to keep this as a set of strings (as `Computed values are not permitted in an enum with string valued members.`). Let me know if there's another way.